mb/mb/testmirror.py

```diff
@@ -155,7 +161,7 @@ def make_probelist(mirrors, filename, url_type='http', get_digest=False, get_con
     The Sample instances are used to hold the probing results.
     """
     from mb.util import Sample
-    if url_type == 'http':
+    if url_type == 'http' or url_type == 'https':
```

Comment:
thinking about it ... you probably want even a different DB column for https url.

Reply:
This PR is aimed to just let `mb` tool accept https. According to Lars https already works if inserted directly to DB